utils/nctl/docs/commands-ctl.md

```diff
@@ -84,40 +84,40 @@ Attempts to rotate a network's validator set - used to verify joining/leaving sc
 nctl-rotate
 ```
 
-### nctl-start node={X:-all} loglevel={Y:-($RUST_LOG | debug)}
+### nctl-start node={X:-all} hash=Y loglevel={Z:-($RUST_LOG | debug)}
 
-Starts node Y, if Y=all then all nodes are started.  
+Starts node X, if X=all then all nodes are started.  If trusted hash is specified then it is injected into node's config prior to spin-up.
 
 ```
 nctl-start
 
 nctl-start node=all loglevel=debug (same as above)
 
-nctl-start node=3 loglevel=info
+nctl-start node=3 hash=<A-TRUSTED_HASH> loglevel=info
 ```
 
-### nctl-start-after-n-blocks node={X:-all} offset={Y:-1} loglevel={Z:-($RUST_LOG | debug)}
+### nctl-start-after-n-blocks node={W:-all} offset={X:-1} hash=Y loglevel={Z:-($RUST_LOG | debug)}
```

Comment:
I am not sure we will be able to use this command with `hash=…` in practice. It's not possible to know the trusted hash to sync with in advance to starting the network.